app/views/users/index.html.erb

```diff
@@ -2,7 +2,25 @@
   Leden
 <% end %>
 
+<% content_for :subtitle do %>
+  <div class="flex items-center text-sm text-gray-400">
+    <%= heroicon('user', class_name: "flex-shrink-0 mr-1.5 h-5 w-5 text-gray-400") %>
+    <%= @leden.count %> leden
+  </div>
+  <% if @aspiranten.any? %>
+    <div class="flex items-center text-sm text-gray-400">
+      <%= heroicon('user-add', class_name: "flex-shrink-0 mr-1.5 h-5 w-5 text-gray-400") %>
+      <%= @aspiranten.count %> aspiranten
+    </div>
+  <% end %>
+  <div class="flex items-center text-sm text-gray-400">
+    <%= heroicon('academic-cap', class_name: "flex-shrink-0 mr-1.5 h-5 w-5 text-gray-400") %>
+    <%= @oudelullen.count %> oud-leden
+  </div>
+<% end %>
+
 <% content_for :options do %>
+  <%= link_to "Aspirant uitnodigen", new_user_invitation_path, class: button_classes if current_user&.admin? %>
```

Comment:
Word deze knop hier niet te zichtbaar? Het grootste deel van de tijd is deze totaal irrelevant omdat er heel weinig accounts aangemaakt moeten worden. Hoe hij nu verstopt zit op de aspiranten pagina vind ik eigenlijk beter.

Reply:
Zit wat in. Anderzijds is het nu vreemd dat een lege tab zo prominent staat (nog voor oud-leden), dus daar komt het idee vandaan om hem weg te halen. We hebben er toch niets aan als er geen aspiranten zijn. Dan heb je vanzelf het probleem dat die knop ergens heen moet, dus dan maar naar de plek die we toch al gebruiken voor knoppen